scaling/iam.tf

```diff
@@ -0,0 +1,27 @@
+resource "aws_iam_role" "ecs-autoscale-role" {
+  name = "ecs-scale-${terraform.env}-${var.cluster_name}"
```

Comment:
@ThomasVdBerge you didn't fix the naming of the iam resources with your last commit, so my initial remark is still valid. ;-)